packages/widget/src/components/transfer-status/transfer-status.ts

```diff
@@ -0,0 +1,57 @@
+import { html, type HTMLTemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { BaseComponent } from '../base-component/base-component';
+import { greenMark, networkIconsMap } from '../../assets';
+import { styles } from './styles';
+
+@customElement('sygma-transfer-status')
+export class TransferStatus extends BaseComponent {
+  static styles = styles;
+
+  @property({ type: String }) from: string = '';
+
+  @property({ type: String }) to: string = '';
+
+  @property({ type: String }) amount: string = '';
+
+  @property({ type: String }) tokenSymbol: string = '';
+
+  @property({ type: String }) explorerLinkTo: string = '';
```

Comment:
maybe better to take txId/txHash?

Reply:
do you mean, renaming this as txHash? How do we want to build the url to use? I was just thinking to just pass that.

Reply:
We need to have url constants and depending on env build URL?